app/src/main/java/com/example/peterchu/watplanner/home/HomeFragment.java

```diff
@@ -176,7 +176,7 @@ public void showExportCalendarDialog() {
             public void onClick(DialogInterface dialog, int which) {
                 // TODO: Replace empty list with the configuration of course components for courses
                 // that the user selected/SAT solver came up with
-                homePresenter.onExportCoursesToCalendar(mCourseSchedule);
+                HomeFragment.this.exportCalendar();
```

Comment:
nope, it's in an anonymous inner function, so the "this" keyword would refer to the DialogInterface OnClickListener